8514 - Load story template sections on scroll for C&U page only

anjenkin opened the pull request:
https://federal-spending-transparency.atlassian.net/browse/DA-8514

brianpetway commented on src/styles/progress.scss:
```diff
@@ -1,8 +1,46 @@
 
 .progress {
-  color: #512da8;
+  //color: #512da8;
```
Can this be removed if it isn't needed?

anjenkin replied:
this is actually necessary so added this back

brianpetway commented on src/styles/progress.scss:
```diff
@@ -1,8 +1,46 @@
+  width: 100%;
+  border-radius: 5px;
+  margin-top: 10px;
+  height: 300px;
```
There are two heights on this class.

brianpetway commented on src/styles/progress.scss:
```diff
@@ -1,8 +1,46 @@
+  height: 75px;
+  margin: 0 0 10px 0;
+  border-radius: 5px;
+  height: 100%;
```
There are two heights on this class.

anjenkin replied:
fixed

brianpetway commented on src/pages/colleges-and-universities/index.jsx:
```diff
@@ -104,11 +110,115 @@ const footnotesProp = [
+					if (entry.isIntersecting && ratio > 0.5 && ratio < 1 && inView) {
+						setActiveSectionIndex(sectionsByName.indexOf(section));
+
+						// this doesn't work for some reason
```
Is this comment still needed?

anjenkin replied:
I update the comment to be more descriptive.  You can update the state here but the updated state isn't accessible from inside the intersection observer for some reason

brianpetway commented:
A few very small things noted inline above.
I can't tell what changes were made in mapbox.jsx bc most of the file is showing as changed bc of reformatting.
Section 2 and the mapbox component still seem to be taking a long time to load when you scroll down to them, but maybe it will be better in QAT.

anjenkin commented:
Ready for re-review.  For mapbox, I removed lines 8 - 10 (the condition where we're checking if the props are not empty) and updated line 291 so the useEffect only runs once.  This was causing the map to reset
